conans/client/conf/__init__.py

```diff
@@ -92,10 +92,8 @@
                       llvm, ClangCL, v143]
             cppstd: [None, 14, 17, 20, 23]
         msvc:
-            version: ["19.0",
-                      "19.1X", "19.10", "19.11", "19.12", "19.13", "19.14", "19.15", "19.16",
-                      "19.2X", "19.20", "19.21", "19.22", "19.23", "19.24", "19.25", "19.26", "19.27", "19.28", "19.29",
-                      "19.3X", "19.30"]
+            version: [190, 191, 192, 193]
+            update: [None, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
See above comment. VS 16.8 and 16.9 both have "update 8", yet those are link-incompatible. Same stands for VS 16.10 and 16.11 (both are "update 9"). You need to handle that case as well.